test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -44,6 +46,8 @@ def convert_to_onnx(model, input=None, opset_version=9, example_outputs=None,
                        fixed_batch_size=fixed_batch_size, training=training,
                        onnx_shape_inference=onnx_shape_inference,
                        use_new_jit_passes=use_new_jit_passes)
+    export_time = time.perf_counter() - export_time
+    print('Time to export: ', export_time)
```

Comment:
Do we need this in the CI tests?

Reply:
i feel it's a nice to have, makes it easier to monitor how changes affect export time. Perhaps can consider adding something like this to export api in the future.